### PR TITLE
Run Oozie in High Availability Mode

### PR DESCRIPTION
**Pull request by adam-stasiak**
This commit solve issue from #211.

### CODE REVIEW

File: oozie/oozie.sh [hunk 1]
```diff
@@ -87,14 +79,48 @@ EOF
   tar -xvzf /usr/lib/oozie/oozie-sharelib.tar.gz
   ln -s share /usr/lib/oozie/share
   hadoop fs -mkdir -p /user/oozie/
-  hadoop fs -put share/ /user/oozie/
+  hadoop fs -put -f share/ /user/oozie/
+  local additional_nodes=$(/usr/share/google/get_metadata_value attributes/dataproc-master-additional | sed 's/,/\n/g' | wc -l)
+  if [[ ${additional_nodes} -ge 2 ]]; then
+    echo 'Starting configuration for HA'
+    local servers=$(cat /usr/lib/zookeeper/conf/zoo.cfg | grep 'server.' | sed 's/server.//g' | sed 's/:2888:3888//g' | cut -d'=' -f2- | sed 's/\n/,/g' | head -n 3 |sed 's/$/:2181,/g' | xargs -L3 | sed 's/.$//g')
```
Comment (karth295) [inline oozie/oozie.sh]: Please comment on what's happening here, particularly the sed commands. Also consider breaking it up into multiple lines :)

File: oozie/oozie.sh [hunk 2]
```diff
@@ -87,14 +79,48 @@ EOF
...
-  # Clean up temporary fles
-  rm -rf ext-2.2 ext-2.2.zip core-site-patch.xml share oozie-sharelib.tar.gz
+<property>
+   <name>oozie.zookeeper.connection.string</name>
+   <value>${servers}</value>
+</property>
+
+<property>
+    <name>oozie.base.url</name>
+    <value>http://my.loadbalancer.$(hostname):11000/oozie</value>
```
Comment (karth295) [inline oozie/oozie.sh]: Since we're not actually using a load balancer, is this necessary?

File: oozie/oozie.sh [hunk 3]
```diff
@@ -87,14 +79,48 @@ EOF
...
+</property>
+</configuration>
+EOF
+sed -i '/<\/configuration>/e cat oozie-site-patch.xml' \
```
Comment (karth295) [inline oozie/oozie.sh]: Check out `bdconfig merge_configurations`. E.g. https://github.com/GoogleCloudPlatform/bdutil/blob/master/libexec/configure_hadoop.sh#L135

File: oozie/oozie.sh (hunk 2 above)
Comment (adam-stasiak) [reply oozie/oozie.sh]: Removed in last commit

**Comment by adam-stasiak** [comment]
@karth295 Please review current state.

**Comment by adam-stasiak** [comment]
@karth295 Can it be merged?

**Comment by karth295** [comment]
I made a separate PR to improve the README instructions: #283 , but this PR lgtm.

**Pull request merged**